test/config/index.spec.ts

```
@@ -0,0 +1,19 @@
+import { getEnvironmentValue, CDN_HOST } from "../../src/config";
+
+describe('Config test suite', () => {
+  it('should check if CDN_HOST env is returned correctly and fetched from user environment', () => {
+    const test_cdn_host = getEnvironmentValue("CDN_HOST");
+    expect(test_cdn_host).toEqual(CDN_HOST);
+  });
+
+  it('should thrown an error when passing anyNonExistingEnv to getEnvironmentValue()', () => {
```

V Minor - 'should throw an error...'

---

Will do, thanks